src/middlewared/middlewared/plugins/smb.py

```diff
@@ -415,6 +415,10 @@ async def configure(self, job, create_paths=True):
         )
         await ds_job.wait()
 
+        # Ensure that winbind is running once we configure SMB service
+        if not await self.middleware.call('service.started', 'idmap'):
+            await self.middleware.call('service.start', 'idmap')
```

Comment:
Should this be propagated to other controller on HA systems? Same for the `service.restart cifs` line a few lines below this?